src/cancel-reminders/lib/db.ts

```diff
@@ -0,0 +1,46 @@
+import { Pool, QueryConfig, QueryResult } from 'pg';
+import { runWithLogging } from '../../lib/db';
+import { Cancellation } from '../lambda/models';
+
+interface CancelQueryResult {
+	oneOffQueryResult: QueryResult;
+	recurringQueryResult: QueryResult;
+}
+
+export async function cancelPendingSignups(
+	cancellation: Cancellation,
+	pool: Pool,
+): Promise<CancelQueryResult> {
```

Comment:
This is slightly clunky, but you're not allowed to do two updates in a single request :(